### ISSUE TITLE
Default color contrast for FloatingActionButton.extended too low

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#19891]
(This issue was found on the Demo page for the Material Search widget in the gallery)

The contrast ratio between text and background of the FloatingActionButton.extended is to low with the default theme. It's 2.24 and should be at least 3.0

Current background color: `#13B9FD`
Current foreground color: `#FFFFFF`

**Comment by pedromassangocode** [comment]
I checked the two variables and can confirm that the default values in the current `dev` channel is a follow:

background color: `#2196f3`
foreground color: `#FFFFFF`

Tested with the default Flutter counter app.

https://github.com/flutter/flutter/blob/7736f3bc90270dcb0480db2ccffbf1d13c28db85/packages/flutter/lib/src/material/floating_action_button.dart#L452-L457
